lib/containers/runtime.pm

```diff
@@ -32,22 +34,71 @@ sub _rt_script_output {
     return script_output($self->runtime . " " . $cmd, @args);
 }
 
-=head2 build
+=head2 create_container($image, $name, [%args])
+
+Creates a container.
+C<image> the name of the image we create the container from.
+C<name> is the given name of the created container.
+C<args> can be anything you want to run in the container, similar to C<run>. for 
+instance: C<docker create -it tumbleweed bash>
+
+=cut
+sub create_container {
+    my ($self, $image, $name, $args) = @_;
+    die 'wrong number of arguments' if @_ < 3;
+    $self->_rt_assert_script_run("container create --name $name $image $args", 300);
+    record_info "$name container created", "";
+}
+
+=head2 start_container($image_name)
+
+Starts container named C<image_name>.
+
+=cut
+sub start_container {
+    my ($self, $image_name) = @_;
+    die 'wrong number of arguments' if @_ < 2;
+    $self->_rt_assert_script_run("container start $image_name");
+    record_info "$image_name container started", "";
+}
+
+=head2 halt_container($container_name)
+
+Blocks a container until exits.
+C<container_name> which runs. 
+https://docs.docker.com/engine/reference/commandline/wait/
+
+=cut
+sub halt_container {
+    my ($self, $container_name) = @_;
+    die 'wrong number of arguments' if @_ < 2;
+    $self->_rt_assert_script_run("wait $container_name");
+    record_info "$container_name container is blocked.", "";
+}
+
+
+=head2 build($dockerfile_path, $container_tag, [%args])
 
 Build a container.
```

Comment:
```suggestion
Build a container from a Dockerfile.
```

I think this added context is useful, especially when not dealing with `docker` as runtime.

Reply:
Done